admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -196,7 +212,9 @@ export const ApplicationPage: React.FunctionComponent = () => {
                             <Tr key={ e.id }>
                                 <Td>{ e.displayName }</Td>
                                 <Td>{ e.name }</Td>
-                                <Td>{ e.description }</Td>
+                                <Td>
+                                    <ListSystemBehaviorGroups />
```

Comment:
I would need to also filter this by bundle and event type.. to only list the current associated behaviors.

Reply:
You could pass the bundle id to the component to only request those components.